README.md

```diff
@@ -162,16 +162,72 @@ res, err := c.ScrapeGoogleSearch(
 	"adidas",
 	&serp.GoogleSearchOpts{
 		Parse: true,
-		Context: []func(serp.ContextOption){
-			serp.ResultsLanguage("en"),
-			serp.Filter(1),
-			serp.Tbm("isch"),
-			serp.LimitPerPage([]serp.PageLimit{{Page: 1, Limit: 1}, {Page: 2, Limit: 6}}),
+		Context: []func(oxylabs.ContextOption){
+			oxylabs.ResultsLanguage("en"),
+			oxylabs.Filter(1),
+			oxylabs.Tbm("isch"),
+			oxylabs.LimitPerPage([]serp.PageLimit{{Page: 1, Limit: 1}, {Page: 2, Limit: 6}}),
 		},
 	},
 )
 ```
 
+### Parse instructions
+
+SDK allows supports [custom parsing](https://developers.oxylabs.io/scraper-apis/custom-parser). 
+There are 2 ways to provide `parsing_instructions` `_fns`:
+
+```go
+package main
+
+import (
+	"fmt"
+	"github.com/mslmio/oxylabs-sdk-go/ecommerce"
+	"github.com/mslmio/oxylabs-sdk-go/oxylabs"
+)
+
+func main() {
+	const username = "username"
+	const password = "password"
+
+	// Initialize the SERP push-pull client with your credentials.
+	c := ecommerce.InitAsync(username, password)
+
+	ch, err := c.ScrapeUniversalUrl(
+		"https://example.com",
+		&ecommerce.UniversalUrlOpts{
+			Parse: true,
+			ParseInstructions: &map[string]interface{}{
```

Comment:
If `_fns` is always required as the root of each key like `title`/`second_paragraph`/etc, can't we simplify further and let the user just give an array of parse instruction objects, each of which takes a name (like `title`), and then a list of function objects of type `oxylabs.Fn`? You can then construct the rest internally.

Or are there more variations?

Reply:
The problem is with pipelines which are very dynamic https://developers.oxylabs.io/scraper-apis/custom-parser/parsing-instruction-examples (see Example 3). Since it might have n-elements, we might need custom marshaling, which I don't think is worth it in this case.

Or at least at the moment I can't see a way to simplify it without introducing additional complexity.